node/src/config.rs

```diff
@@ -64,6 +64,20 @@ impl MpcConfig {
             participants,
         })
     }
+
+    /// A leader – participant with the lowest id.
+    /// Used to calculate who's going to create a channel for a task.
+    pub fn is_leader(&self) -> bool {
```

Comment:
nit: maybe name this "is_leader_for_keygen" or something like that. The problem is that the leader has a different definition in the case of triple/presig/sig generations.